### PR TITLE
Add rectify_image function

### PR DESCRIPTION
**Pull request by iory**
## Example
```
$ python examples/rectify_image.py
```

image

### CODE REVIEW

File: examples/rectify_image.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    depth_to_rgb_transform = np.load(
+        './data/depth_to_rgb_transformation.npy')
+
+    depth_cameramodel = cameramodels.PinholeCameraModel.from_yaml_file(
+        './data/k4a_depth_camera_info.yaml')
+    bgr_cameramodel = cameramodels.PinholeCameraModel.from_yaml_file(
+        './data/k4a_rgb_camera_info.yaml')
+    depth_img = cv2.imread('./data/k4a_depth_image_raw.png',
+                           cv2.IMREAD_ANYDEPTH)
```
Comment (kosuke55) [inline examples/rectify_image.py]: How about writing like
```
current_dir = osp.dirname(osp.abspath(__file__))
depth_to_rgb_transform = np.load(
    osp.join(current_dir, 'data/depth_to_rgb_transformation.npy'))
```
to run from outside the example directory?

Comment (iory) [reply examples/rectify_image.py]: I agree.
Thanks.

Comment (kosuke55) [reply examples/rectify_image.py]: OK, Thanks

**Comment by kosuke55** [comment]
I checked it. I think it's good.

**Comment by iory** [comment]
Thanks!

**Review by kosuke55 — APPROVED** [review]

**Pull request merged**